OpenSim/Simulation/Model/Force.h

```diff
@@ -47,14 +47,18 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Force, ModelComponent);
 //==============================================================================
 // PROPERTIES
 //==============================================================================
-    /** A Force element is active (enabled) by default. **/
-    OpenSim_DECLARE_PROPERTY(isDisabled, bool,
-        "Flag indicating whether the force is disabled or not. Disabled means"
+    /** A Force element is active (enabled) by default.
+    NOTE: Prior to OpenSim 4.0, this property was named **isDisabled**.
+          If **isDisabled** is **true**, **appliesForce** is **false**.
+          If **isDisabled** is **false**, **appliesForce** is **true**.          */
+    OpenSim_DECLARE_PROPERTY(appliesForce, bool,
+        "Flag indicating whether the force is applied or not. Value of false means"
         " that the force is not active in subsequent dynamics realizations.");
```

Comment:
The new comment above is very clear. In the dev meeting, I think we decided to also put a brief note inside the XML comment as well, so that the change would appear in XML files. Did you check how the doxygen block looks in the generated HTML? There's a chance that doxygen blocks for properties don't show up nicely (because of the macro).